index.js

```diff
@@ -75,7 +61,16 @@ export default class ClapprStats extends ContainerPlugin {
 
   stopReporting() {
     this._buildReport()
+
     clearInterval(this._intervalId)
+    this._newMetrics()
+
+    this.container.off(Events.CONTAINER_TIMEUPDATE, this.onContainerUpdateWhilePlaying)
```

Comment:
I'm in favor of [`stopListening`](https://github.com/clappr/clappr/blob/master/src/base/events.js#L157) that usually container [calls](https://github.com/clappr/clappr/blob/master/src/base/container_plugin.js#L36) it during its destruction.  This is the way to un-listen all events.

This code should work: (it really should work otherwise maybe we need to fix Clappr):
```javascript
this.stopListening();
this.bindEvents();
```